index.js

```diff
@@ -244,29 +244,51 @@ const create = (win, options) => {
 			*/
 			menu.popup(electron.remote ? electron.remote.getCurrentWindow() : win);
 		}
-	});
+	};
+
+	webContents(win).on('context-menu', onContextMenu);
+	return () => webContents(win).removeListener('context-menu', onContextMenu);
```

Comment:
I prefer not using an inline arrow function when there's no return value. It just makes the code harder to read. Applies in other places too.

Reply:
Returning a named `function dispose() { ...` is clearly the better option, also when debugging etc.